mod-kitpvp/src/main/java/gg/solarmc/loader/kitpvp/OnlineKitPvp.java

```diff
@@ -73,4 +94,12 @@ public int currentDeaths() {
     public int currentAssists() {
         return assists;
     }
+
+    public int currentCurrentKillstreak() {
+        return currentKillstreaks;
+    }
+
+    public int currentHighestKillstreak() {
+        return highestKillstreaks;
+    }
```

Comment:
Remember to add an experience method too

Reply:
I forgot? aghhhhh!